composer.json

```
@@ -25,7 +25,7 @@
     },
     "require-dev": {
         "codeception/module-asserts": "^3.0",
-        "codeception/module-doctrine2": "^3.0",
+        "codeception/module-doctrine": "^3.0",
```

```suggestion
        "codeception/module-doctrine": "^3.1",
```

This is not really relevant, but we could raise the min version because there is no version 3.0 for `codeception/module-doctrine`.